modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/SelectObjectDefinition.tsx

```diff
@@ -68,6 +68,7 @@ export default function SelectObjectDefinition({
 			disabled={disabled}
 			emptyStateMessage={Liferay.Language.get('no-objects-were-found')}
 			error={error}
+			id="select-object-definition"
```

Comment:
Hi @Artartur. Please, rename the id to `objectRelationshipSelectObjectDefinition`.

Reply:
Done!